package.json

```diff
@@ -149,7 +149,7 @@
     "netlify-redirect-parser": "^14.1.1",
     "netlify-redirector": "^0.4.0",
     "node-fetch": "^2.6.0",
-    "node-version-alias": "^1.0.1",
+    "node-version-alias": "^2.0.0",
```

Comment:
Not updated to 3.4.1 which is the latest version as of now since anything >=3.0.0 only supports Node.js >=14.18.0 while our least supported Node.js version is `v14.16.0`

Ref: https://github.com/ehmicky/node-version-alias/blob/main/CHANGELOG.md#300